src/components/Button/Button.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { ColorType } from '../../styles/theme';
+
+export interface ButtonProps {
+  bgColor?: ColorType;
+  color?: ColorType;
+  btnSize?: 'large' | 'mid';
+  onClick?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+}
+
+type StyleType = Pick<ButtonProps, 'color' | 'bgColor' | 'btnSize'>;
+type SizeType = Pick<ButtonProps, 'btnSize'>;
+const Button: React.FC<ButtonProps> = ({ children, color, bgColor, btnSize, ...props }) => {
+  return (
+    <Wrapper color={color} bgColor={bgColor} btnSize={btnSize} {...props}>
+      {children}
+    </Wrapper>
+  );
+};
+
+Button.defaultProps = {
+  btnSize: 'mid',
+};
```

Comment:
defaultlProps 대신 
```suggestion
const Button: React.FC<ButtonProps> = ({ children, color, bgColor, btnSize = 'mid', ...props }) => {
  return (
    <Wrapper color={color} bgColor={bgColor} btnSize={btnSize} {...props}>
      {children}
    </Wrapper>
  );
};
```

defaultProps 대신 걍 버튼 프로퍼티 default value 주는게 더 직관적일듯?

Reply:
음 당장은 좋은거 같은데, 이건 뭔가 inline으로 작성하는 느낌일거같아서
뭔가 여러가지가 추가되면 defaultProps가 더 깔끔하지 않을까..? 그냥 더 깔끔한쪽이 어떤쪽일까 고민..

Reply:
개인적으로는 `defaultProps`는 뭔가 class component의 잔재같은 느낌이고(당시엔 props에 초기값및 interface type을 넣을 방법이 없었으니)
오히려 인라인으로 작성하는게 코드 작성 수고도 줄고 직관적인 것 같은뎅
@JangKyuBum @MrMirror21 의견 주십쇼

Reply:
둘 다 맞는 말이라 취향차 같긴 한데, 사실 준호 형이 말한대로 분리를 하더라도 위에 `ButtonProps` 있는 건 매한가지라 '분리해서 가독성을 높인다' 효과가 빛이 바래는 느낌이야

Reply:
@juno7803 다수결로 defaultProps 없애겠습니다